app/templates/briefs/not_is_supplier_eligible_for_brief_error.html

```diff
@@ -1,43 +1,27 @@
 {% extends "_base_page.html" %}
 
-{% block page_title %}Not eligible for brief – Digital Marketplace{% endblock %}
+{% block page_title %}Not eligible for opportunity – Digital Marketplace{% endblock %}
 
 {% block main_content %}
 
-{% if clarification_question %}
+{% with inhibited_action="ask a question about" if clarification_question else "apply for" %}
```

Comment:
I think I should start putting deliberately controversial things in my PRs so they get attention and get reviewed. Alternatively I'll start giving them clickbait-y titles...